taskcluster/scripts/toolchain/repack-android-sdk-linux.sh

```diff
@@ -1,10 +1,19 @@
 #!/bin/bash
 
-export ANDROID_SDK_ROOT=$MOZ_FETCHES_DIR
+set -ex
 
+. "$(dirname $0)/directories.sh"
+
+export ANDROID_SDK_ROOT="$MOZ_FETCHES_DIR"
 yes | "${ANDROID_SDK_ROOT}/tools/bin/sdkmanager" --licenses
 
+pushd $PROJECT_DIR
+# XXX The Android SDK is fully populated by gradle if it compiles something.
+./gradlew assembleDebug
```

Comment:
I've looked into finding a finer-grained task, but I couldn't. I used https://plugins.gradle.org/plugin/com.dorongold.task-tree to print the dependency tree and none of my trials got gradle download the SDK. That's why I end up compiling one flavor here.